src/main/resources/messages_et.properties

```diff
@@ -13,9 +13,9 @@ membership.email.invitation=Tutvu teiste Tuleva liikmetega
 membership.email.fbgroup=meie Facebooki grupis
 membership.email.capital=Ühistu liikmed saavad veel kuni 31.maini teha vabatahtliku sissemakse liikmekapitali.
 membership.email.capital.link=Loe lähemalt!
-mandate.email.suggestThirdPillar=Järgmisena sea üles kolmas sammas. Sel aastal tehtud sissemaksetelt annab riik sulle tulumaksu juba kevadel tagasi. <a href="https://tuleva.ee/iii-sammas/">Alusta siit.</a>
-mandate.email.suggestSecondPillar=Tee korda ka oma teine sammas ja vali soodsa tasuga fond. <a href="https://tuleva.ee/kuidas-tuua-pension-tulevasse/">Too teine sammas Tulevasse</a>.
-mandate.email.suggestMembership=Tuleva pensionifondide valitseja kuulub inimestele, kes ka ise neis fondides oma tuleviku jaoks raha koguvad. Tuleva pensionifondide kliendina ei ole sul mingit kohustust Tuleva ühistu liikmeks astuda - meiega koos võivad kaasaegses, madalate kuludega fondis pensionit koguda kõik Eesti inimesed. Aga kutsun sind liikmeks tulemist kaaluma, sest kõige rohkem saad Tulevast kasu, olles ise kaasomanik.<br><br><a href="https://tuleva.ee/liikmetele/31-marts-korduma-kippuvad-kusimused/?utm_source=clients&utm_medium=email&utm_campaign=new_client_welcome">Vaata lähemalt, mida Tuleva liikmeks olek tähendab</a>
+mandate.email.suggestThirdPillar=Järgmisena sea üles kolmas sammas. Sel aastal tehtud sissemaksetelt annab riik sulle tulumaksu juba kevadel tagasi. <a href="https://tuleva.ee/iii-sammas/?utm_source=ii_pillar_clients_transfer&utm_medium=email&utm_campaign=welcome_email">Alusta siit.</a>
+mandate.email.suggestSecondPillar=Tee korda ka oma teine sammas ja vali soodsa tasuga fond. <a href="https://tuleva.ee/kuidas-tuua-pension-tulevasse/?utm_source=iii_pillar_clients_transfer&utm_medium=email&utm_campaign=welcome_email">Too teine sammas Tulevasse</a>.
```

Comment:
suggestSecondPillar -> iii_pillar_clients_transfer
suggestThirdPillar -> ii_pillar_clients_transfer ?

Reply:
Source has to be simple -> iii_pillar_clients is enough for me